packages/react-tooltip/src/components/Tooltip/useTooltipStyles.ts

```diff
@@ -40,14 +41,16 @@ const useStyles = makeStyles({
 
   // width and height = arrowHeight * sqrt(2)
   // Update arrowHeight in useTooltip.tsx if this changes
-  arrow: createArrowStyles(8.485),
+  arrowLTR: createArrowStyles({ size: 8.485, dir: 'ltr' }),
```

Comment:
nit: duplicated number

```suggestion
  arrowLTR: createArrowStyles({ size: arrowSize, dir: 'ltr' }),
```